packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -359,7 +360,7 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
         //         Lib_OVMCodec.QueueElement memory el = getQueueElement(nextQueueIndex);
         //         require(
         //             el.timestamp + forceInclusionPeriodSeconds < block.timestamp,
-        //             "Queue transactions cannot be submitted during the sequencer inclusion period."
+        //           "Queue transactions cannot be submitted during the sequencer inclusion period."
```

Comment:
I was confused by this but I assume it's because you can't add another `//` to ignore the line length since this already a comment (?)

Reply:
Although maybe you could do it the same way you did below by disabling it for the whole block.